build.yaml

```diff
@@ -103,6 +103,22 @@ build:
         python setup.py build_ext --inplace --no-cython
       fi
 
+      echo "Installing simulacron"
```

Comment:
It could be easier/faster to do:

```bash
wget https://github.com/datastax/simulacron/releases/download/0.5.0/simulacron-standalone-0.5.0.jar
```

Also we could bake a simulacron jar into our jenkins image if that makes it easier too.